ConsoleApp_Task1/Program.cs

```diff
@@ -82,44 +34,68 @@ public static double ReturnRadiusOrSide()
                             {
                                 InputString = InputString.Replace(",", ".");
```

Comment:
This validation doesn't work on the system with ',' as a decimal delimiter. In order to make your program work correctly on all systems, you can adjust your validation.
I propose to make it an optional task for you